model/directory_status.py

```diff
@@ -4,6 +4,28 @@ class DirectoryStatus:
     contain the relevant data.
     """
 
+    # States that are valid for the status field
+
+    # New runs, ready to upload
+    NEW = 'new'
+    # Used when a run directory does not have the base requirements to act as a run
+    # Never written to a status file
+    INVALID = 'invalid'
+    # Parsing/Upload has started/partially completed for this run
+    PARTIAL = 'partial'
+    # Parsing/Upload has stopped because of some error.
+    ERROR = 'error'
+    # Parsing/Upload has completed successfully
+    COMPLETE = 'complete'
```

Comment:
Is this list of statuses in the docs somewhere?  if not it should be.  Same content above is fine.

Reply:
It was, but I have now updated it to better reflect how those statuses should be used in practice (not using the strings raw, but using the final variables).